query/search.js

```diff
@@ -160,6 +162,10 @@ function getQuery(vs) {
 
 }
 
+function forceLibpostal(params) {
+  return _.isSet(params, 'force_libpostal');
```

Comment:
I don't think this is doing what you're expecting.  `isSet` just returns whether the lone parameter is actually a `Set`.  You probably want `_.get(params, 'force_libpostal', false)`.